### PR TITLE
Feature/add edit post screen

### PR DESCRIPTION
**Pull request by WalterGoldenOwl**
Description Pull Request:
## Todo
- Code UI and logic Add/Edit Post Screen
- Apply Logic for Connection Tab

## Task
Link of task: 

1. https://trello.com/c/FVQkvDAi
2. https://trello.com/c/27m5OnGY
3. https://trello.com/c/SShyXM83

## Screenshot
Screenshot_1696582049
Screenshot_1696582056
Screenshot_1696582062

## Video

https://github.com/dustin-goldenowl/jobspot/assets/144195318/80ff5737-db0a-4d63-82f2-51edeefc1446

https://github.com/dustin-goldenowl/jobspot/assets/144195318/e848faa2-ca34-4609-b12d-ddac1c838fa8

### CODE REVIEW

File: lib/src/presentations/add_post/screen/add_post_view.dart [hunk 1]
```diff
@@ -0,0 +1,209 @@
...
+          return SizedBox(
+            height: 100,
+            child: ListView.separated(
+              scrollDirection: Axis.horizontal,
+              physics: const BouncingScrollPhysics(),
+              itemBuilder: (context, index) {
+                String image = context.read<AddPostCubit>().state.images[index];
+                return Stack(
+                  children: [
+                    if (image.isLink)
```
Comment (dustin-goldenowl) [inline lib/src/presentations/add_post/screen/add_post_view.dart]: Should be separate a new widget

File: lib/src/presentations/connection/cubit/connection_cubit.dart [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+              user: listUser.firstWhere((element) => element.id == e.owner),
+            ),
+          )
+          .toList();
+      _limit = limit < documents.count ? limit : documents.count;
+      emit(state.copyWith(posts: posts, isMore: limit < documents.count));
+    });
+  }
+
+  Future<List<UserModel>> getListUser(List<PostModel> datas) async {
```
Comment (dustin-goldenowl) [inline lib/src/presentations/connection/cubit/connection_cubit.dart]: Don't use Firebase in cubit. Let's define another file

File: lib/src/presentations/add_post/screen/add_post_view.dart (hunk 1 above)
Comment (WalterGoldenOwl) [reply lib/src/presentations/add_post/screen/add_post_view.dart]: I fixed it

File: lib/src/presentations/connection/cubit/connection_cubit.dart (hunk 2 above)
Comment (WalterGoldenOwl) [reply lib/src/presentations/connection/cubit/connection_cubit.dart]: I fixed it

**Pull request merged**